cogeo_mosaic/backends/base.py

```diff
@@ -57,29 +59,36 @@ class BaseBackend(BaseReader):
     ] = attr.ib(default=Reader)
     reader_options: Dict = attr.ib(factory=dict)
 
-    geographic_crs: CRS = attr.ib(default=WGS84_CRS)
-
-    # TMS is outside the init because mosaicJSON and cogeo-mosaic only
-    # works with WebMercator for now.
+    # TMS Should be the same as the Mosaic's TMS
     tms: TileMatrixSet = attr.ib(init=False, default=WEB_MERCATOR_TMS)
```

Comment:
This should be updated (I reverted the change).

The TMS is defined by the Mosaic, but this is only supported in mosaicjson 0.0.3